docs/configuration/alertmanager-integration/victoria-metrics.rst

```diff
@@ -8,26 +8,49 @@ You will need to configure two integrations: one to send alerts to Robusta and a
 Send Alerts to Robusta
 ============================
 
-To configure, edit AlertManager's configuration:
+Add the following to your Victoria Metrics Alertmanager configuration (e.g., Helm values file or VMAlertmanagerConfig CRD):
+
+.. code-block:: yaml
+
+    receivers:
+        - name: 'robusta'
```

Comment:
I think the indentation here might be wrong
should `-name` be 2 points to the left?